src/main/kotlin/com/autonomousapps/internal/ManifestParser.kt

```diff
@@ -18,8 +19,8 @@ internal class ManifestParser(
     val packageName: String,
     /** The value of the `android:name` attribute. May be empty. */
     val applicationName: String,
-    /** The values of the `android:theme` attributes of application and activities. May be empty. */
-    val themes: Set<String>,
```

Comment:
I merge the `themes` inside a generic `attrRefs`.
Also, I parsed it directly as `AndroidResSource.AttrRef` to make tests more expressive (instead of in `explodeAndroidResSource.kt`).